app/serializers/api/v1/task_serializer.rb

```diff
@@ -4,14 +4,39 @@ module Api
   module V1
     class TaskSerializer < ActiveModel::Serializer
       type :tasks
-      attributes :id, :name, :started_on, :due_date, :description, :state, :archived
+      attributes :id, :name, :started_on, :due_date, :description, :state, :archived, :status_id, :status_name,
+                 :prev_status_id, :prev_status_name, :next_status_id, :next_status_name
       has_many :output_tasks, key: :outputs,
                             serializer: TaskSerializer,
                             class_name: 'MyModule'
       has_many :input_tasks, key: :inputs,
                              serializer: TaskSerializer,
                              class_name: 'MyModule'
 
+      def status_id
+        object.my_module_status_id
+      end
+
+      def status_name
+        object.my_module_status.name
+      end
+
+      def prev_status_id
```

Comment:
You can use belongs_to for previous and next status with its serializer...?

Reply:
It does not belong to my_module

Reply:
But status `belongs_to`...